install.sh

```diff
@@ -258,7 +258,37 @@ resume_bamboo_server() {
         echo "Please enter password of the Bamboo '${ADMIN_USERNAME}' user: "
         read -s ADMIN_PASSWORD
       fi
+
       bamboo_url=$(terraform output | grep '"bamboo" =' | sed -nE 's/^.*"(.*)".*$/\1/p')
+
+      status_url="${bamboo_url}/rest/api/latest/status"
+      local RESULT=$(curl -s "${status_url}")
+      log "status endpoint result ${RESULT}"
+      if [[ "x${RESULT}" == *"RUNNING"* ]]; then
+        log "Bamboo server is already ${RESULT}, skip resuming."
+        return
+      elif [[ "x${RESULT}" == *"PAUSE"* ]]; then
+        log "paused, do nothing"
+      else
+        log "neither RUNNING nor PAUSED, sleep for 10 mins to debug."
+        local STATUS_ENDPOINT_RESULT
+        sleep 1m
+        STATUS_ENDPOINT_RESULT=$(curl -s "${status_url}")
+        log "status endpoint result after 1 min ${STATUS_ENDPOINT_RESULT}"
+        sleep 1m
+        STATUS_ENDPOINT_RESULT=$(curl -s "${status_url}")
+        log "status endpoint result after 2 min ${STATUS_ENDPOINT_RESULT}"
+        sleep 1m
+        STATUS_ENDPOINT_RESULT=$(curl -s "${status_url}")
+        log "status endpoint result after 3 min ${STATUS_ENDPOINT_RESULT}"
+        sleep 1m
+        STATUS_ENDPOINT_RESULT=$(curl -s "${status_url}")
+        log "status endpoint result after 4 min ${STATUS_ENDPOINT_RESULT}"
+        sleep 1m
+        STATUS_ENDPOINT_RESULT=$(curl -s "${status_url}")
+        log "status endpoint result after 5 min ${STATUS_ENDPOINT_RESULT}"
```

Comment:
Hi @yzha645 this could probably be DRY'ed up a bit, there is a lot of duplication. This block could probably be extracted into its own method that performs a while loop where a counter is incremented to `5`.